app/views/content/subjects/physics/_article.html.erb

```diff
@@ -0,0 +1,168 @@
+        <ul class="vertical-tags">
+          <li class="tag">Energy</li>
+          <li class="tag">Motion and forces</li>
+          <li class="tag">Electricity and electromagnetism</li>
+          <li class="tag">Waves and matter</li>
+        </ul>
+      <p>
+        14 to 16 year olds (key stage 4):
+      </p>
+       <ul class="vertical-tags">
+        <li class="tag">Energy, forces and wave motion</li>
+        <li class="tag">Electricity, magnetism and electromagnetism</li>
+        <li class="tag">The structure of matter</li>
+        <li class="tag">Atomic structure and space physics</li>
+      </ul>
+      <p>
+        16 to 18 year olds (key stage 5):
+      </p>
+       <ul class="vertical-tags">
+        <li class="tag">Fields, vectors and scalars</li>
+        <li class="tag">Mechanics and mechanical properties of matter</li>
+        <li class="tag">Electric circuits and waves</li>
+        <li class="tag">Matter, quantum and nuclear physics</li>
+      </ul>
+    <p>
+      <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study">Read the national curriculum for more information about physics lessons</a>.
+    </p>
+
+    <p>You can also search for past exam papers online to see the type of questions that have been covered.</p>
+
+    <p>
+      You may also need to teach other subjects, particularly for 11 to 16 year olds and in some cases for post-16. This will vary depending on the individual schools.
+    </p>
+    <p><a href="https://www.ogdentrust.com/teacher-support/training-to-be-a-teacher/">Find out more about becoming a physics teacher from the Ogden Trust</a>, a charitable trust which promotes the teaching and learning of physics. </p>
+  </section>
+   <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "It's inspiring getting 'wow, that's how it works' from pupils as they grasp a subject that they thought was hard.",
+    name: "Jane Doran, physics and robotics teacher, West Midlands",
+    background: "grey",
+    large: true
+  ) %>
+</section>
+
+ <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithOneImageComponent.new(
+      heading: "Next steps into teacher training",
+      colour: "yellow",
+      image_path: "static/content/hero-images/0006.jpg",
+      image_alt: "Classroom full of pupils.",
+    ) %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Help to become a physics teacher</h3>
+
+    <p>
+      Our experienced teacher training advisers can guide you through everything you need to know to apply for physics teacher training, free of charge. They can help you to:
+    </p>
+    <ul>
+      <li>understand more about physics courses</li>
+      <li>answer questions about fees and funding</li>
+      <li>make an application for a physics course</li>
+      <li>prepare for your teacher training interview</li>
```

Comment:
just wondering if 4 bullets is perhaps too much? esp given we're taking them to the content page about advisers where they'll get all of this, too? (I am unnecessarily biased over the rule of 3 though!!)

Reply:
Yes good point, 4 is probably over the top!